src/main/scala/org/neo4j/spark/utils/Neo4jUtils.scala

```diff
@@ -0,0 +1,16 @@
+package org.neo4j.spark.utils
+
+import org.neo4j.driver.{Driver, Session}
+
+object Neo4jUtils {
+
+  def close(driver: Driver, session: Session): Unit = {
+    if (session != null && session.isOpen) {
```

Comment:
wrap with try-catch to guarantee to close the driver even if closing the session fails for some reason,
want to avoid leaks

Reply:
perfect, done